### PR TITLE
feat: add ethereum rpc table component and example

### PR DESCRIPTION
**Pull request by yqrashawn**
## Description

- add rpc table component through `ethereum-rpc-table` package
- add example usage in `docs/en/user-guide/setup.mdx`, this is just an example, the table is not supposed to be there, **do not merge**
- package code at https://github.com/scroll-tech/ethereum-rpc-table

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205686187705236

### CODE REVIEW

File: src/components/RPCTable/RPCTable.astro [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  parent: {
+    type: "L2",
+    chain: "eip155-1",
+    bridges: [
+      {
+        url: "https://scroll.io/bridge",
+      },
+    ],
+  },
+}
```
Comment (yqrashawn) [inline src/components/RPCTable/RPCTable.astro:72]: example data from https://chainlist.org/_next/data/rZLRgM60FTmTVWO5jkIXw/chain/534352.json?chain=534352

File: src/components/RPCTable/RPCTable.astro [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+copy = copy || {
+  "RPC Server Address": "RPC Server Address",
+  Height: "Height",
+  Latency: "Latency",
+  Score: "Score",
+  Privacy: "Privacy",
+  "Add Network": "Add Network",
+  Connect: "Connect",
+  Sorting: "Sorting",
+}
```
Comment (yqrashawn) [inline src/components/RPCTable/RPCTable.astro]: all copy in this component, for i18n

File: src/content/docs/en/user-guide/setup.mdx [hunk 3]
```diff
@@ -31,6 +32,8 @@ The **Add to wallet** links may not be compatible with every wallet. If you are
 | Currency Symbol    | ETH                                                              | ETH                                                                          |
 | Block Explorer URL | [https://sepolia.scrollscan.com](https://sepolia.scrollscan.com) | [https://sepolia.etherscan.io](https://sepolia.etherscan.io)                 |
 
+<RPCTable />
```
Comment (yqrashawn) [inline src/content/docs/en/user-guide/setup.mdx]: Example usage

**please put the table in the appropriate place or delete this before merge**

**Comment by dghelm** [comment]
Question for @zzq0826 and @Turupawn -- Do we want i18n strings to live close to components, or would we prefer everything for a single language to sit in its `translation.json` file?

**Comment by zzq0826** [comment]
> Question for @zzq0826 and @Turupawn -- Do we want i18n strings to live close to components, or would we prefer everything for a single language to sit in its `translation.json` file?

If possible, I prefer to configure it in the` translation.json` file.

**Comment by yqrashawn** [comment]
It's still possible to have i18n strings in [lang]/translation.json with this component.
I've updated the code to relfect that. I also added en and zh translation to json file.

@dghelm @zzq0826

**Comment by yqrashawn** [comment]
To add custom style, you can remove
`import "ethereum-rpc-table/lib/default.css"` and `import "./RPCTable.css"` at https://github.com/scroll-tech/scroll-documentation/blob/6d0db315664e9607cdef0b3587277c7b8562ff0a/src/components/RPCTable/RPCTable.astro#L3-L5

The package adds zero style if these two imports are removed

File: src/components/RPCTable/RPCTable.astro (hunk 1 above)
Comment (dghelm) [reply src/components/RPCTable/RPCTable.astro:72]: This is good for now!

**Review by dghelm — APPROVED** [review]
LGTM -- adjusted css and moved to components preview page.

Might assess followup for mobile, but we'll likely just want to hide it and make a component that alternates display after different breakpoints.

**Pull request merged**